src/components/Button/index.tsx

```diff
@@ -301,14 +301,43 @@ export function ButtonError({ error, ...rest }: { error?: boolean } & ButtonProp
   }
 }
 
-export function ButtonDropdown({ disabled = false, children, ...rest }: { disabled?: boolean } & ButtonProps) {
+export const StyledButtonDropdown = styled(Base)`
+  background-color: ${({ theme }) => theme.bg3};
+  color: ${({ theme }) => theme.primary1};
+  border-radius: 4px;
+  border: none;
+  font-size: 12px;
+
+  &:focus {
+    box-shadow: 0 0 0 1px ${({ theme }) => theme.bg4};
+  }
+  &:hover {
+    box-shadow: 0 0 0 1px ${({ theme }) => theme.bg4};
+  }
+  &:active {
+    box-shadow: 0 0 0 1px ${({ theme }) => theme.bg4};
+  }
+  &:disabled {
+    opacity: 50%;
+    cursor: auto;
+  }
+`
+
+export function ButtonDropdown({
+  expanded = true,
+  disabled = false,
+  children,
+  ...rest
+}: { expanded: boolean; disabled?: boolean } & ButtonProps) {
+  const theme = useContext(ThemeContext)
```

Comment:
```
import useTheme from 'hooks/useTheme'
...
const theme = useTheme()
```
This is better than use useContext. 

Here is some best practices about React Context I read before: https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
Thanks for the catch, I updated it